tests/rbd_mirror/test_image_removal_from_secondary.py

```diff
@@ -0,0 +1,86 @@
+import time
+
+from tests.rbd_mirror.rbd_mirror_utils import rbd_mirror_config
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def test_image_removal_from_secondary(rbd_mirror, pool_type, **kw):
+    try:
+        mirror1 = rbd_mirror.get("mirror1")
+        mirror2 = rbd_mirror.get("mirror2")
+        config = kw.get("config")
+        pool = config[pool_type]["pool"]
+        image = config[pool_type]["image"]
+        imagespec = pool + "/" + image
+        # verify for journal mirroring enable
+        if mirror1.get_mirror_mode(imagespec) == "journal":
+            log.info("journal mirroring is enabled")
+
+        count = 0
+        # Disable and enable journaling on primary image while writing some IOs
+        while count < config.get("repeat_count"):
+            mirror1.image_feature_disable(
+                imagespec=imagespec, image_feature="journaling"
+            )
+            mirror1.benchwrite(imagespec=imagespec, io=config.get("io-total"))
+            mirror1.image_feature_enable(
+                imagespec=imagespec, image_feature="journaling"
+            )
+            time.sleep(60)
+
+            count += 1
+
+        # again disable the journaling feature on primary image
+        mirror1.image_feature_disable(imagespec=imagespec, image_feature="journaling")
+        time.sleep(60)
+
+        # verify removal from secondary
+        if mirror2.image_exists(imagespec):
+            log.info("Image does not exist after disabling journal mirroring")
+            return 0
+        else:
+            log.error(
+                "Image still exist in secondary node even after removing the journal mirroring"
```

Comment:
please make it as secondary cluster?

Reply:
did u mean change is sentence?

Reply:
yes it's not secondary node it's a `secondary cluster`